.github/workflows/ci.yml

```diff
@@ -179,31 +186,144 @@ jobs:
           echo "POLICY_TF_VAR_FILE=${VAR_FILE}" >> $GITHUB_ENV;
       - name: 'Init the terraform infrastructure for mapping'
         run: |-
-          cp -r ${{ env.TF_MODULE_DIR}} ${{ env.MAPPING_TF_MODULE_DIR}}
-          terraform -chdir=${{ env.MAPPING_TF_MODULE_DIR}} init
+          cp -r ${{ env.TF_MODULE_DIR }}/* ${{ runner.temp }}/${{ env.MAPPING_TF_MODULE_DIR }}
+          terraform -chdir=${{ runner.temp }}/${{ env.MAPPING_TF_MODULE_DIR }} init
       - name: 'Apply the infrastructure updates for mapping'
         id: 'tf_apply_mapping'
         run: |-
-          terraform -chdir=${{ env.MAPPING_TF_MODULE_DIR}} apply -auto-approve -var-file=${{env.MAPPING_TF_VAR_FILE}};
+          terraform -chdir=${{ runner.temp }}/${{ env.MAPPING_TF_MODULE_DIR }} apply -auto-approve -var-file=${{ env.MAPPING_TF_VAR_FILE }};
       - name: 'Init the terraform infrastructure for policy'
         run: |-
-          cp -r ${{ env.TF_MODULE_DIR}} ${{ env.POLICY_TF_MODULE_DIR}}
-          terraform -chdir=${{ env.POLICY_TF_MODULE_DIR}} init
+          cp -r ${{ env.TF_MODULE_DIR }}/* ${{ runner.temp }}/${{ env.POLICY_TF_MODULE_DIR }}
```

Comment:
Do you need `/*` here?

Reply:
Becase the `POLICY_TF_MODULE_DIR` is mkdir-ed in a previous step already, so we only want the content in `TF_MODULE_DIE` to be copied. Without `/*` here, after the copy command the file structure would be
```
|- POLICY_TF_MODULE_DIR
   |- xxxx.tfvar
   |- TF_MODULE_DIR
       |- ***.tf
```

Reply:
I got a little bit confused. At first, mu question is we used `cp -r ${{ env.TF_MODULE_DIR}} ${{ env.POLICY_TF_MODULE_DIR}}` and you changed it to `cp -r ${{ env.TF_MODULE_DIR }}/* ${{ runner.temp }}/${{ env.POLICY_TF_MODULE_DIR }}` I understand the change from `${{ env.POLICY_TF_MODULE_DIR}}` to ` ${{ runner.temp }}/${{ env.POLICY_TF_MODULE_DIR }}` as you use ${{ runner.temp }}. Based on my understanding, -r already copies file recursively under ${{ env.TF_MODULE_DIR}}

Reply:
Discussed offline.